test/testsymboldatabase.cpp

```diff
@@ -3209,6 +3210,18 @@ class TestSymbolDatabase : public TestFixture {
         ASSERT_EQUALS(2U, fredAType->classDef->linenr());
     }
 
+    void namespaces5() { // #13967
+        GET_SYMBOL_DB("namespace test {\n"
+                      "  template <int S>\n"
+                      "  struct Test { int x[S]; };\n"
+                      "  const Test<64> test;\n"
+                      "}\n");
+        const Variable *x = db->getVariableFromVarId(2U);
+        ASSERT_EQUALS("x", x->name());
+        const Scope *scope = x->scope();
+        ASSERT_EQUALS("test", scope->nestedIn->className);
```

Comment:
this test will not work well if `scope->nestedIn` is nullptr I want an extra ASSERT to ensure proper error output..